app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt

```diff
@@ -66,6 +73,7 @@ class TransactionListFragment : SafeOverviewBaseFragment<FragmentTransactionList
                     showEmptyState()
                 } else {
                     showList()
+                    binding.transactions.scrollToPosition(0)
                 }
```

Comment:
Doesn't that scroll to the top each time the list is updated?

Reply:
it won't do it by default, so we have to do it manually